ROMFS/px4fmu_common/mixers/vtol_AERT.aux.mix

```diff
@@ -0,0 +1,30 @@
+Mixer for a horizontal format with X-Quad and tractor AERT plane configuration
```

Comment:
I see what you mean. I never realised there was an intentional mapping between the two. TAAER is going to make an interesting filename though.